Source/WebCore/platform/graphics/texmap/GraphicsContextGLTextureMapperANGLE.cpp

```diff
@@ -107,10 +107,16 @@ GraphicsContextGLANGLE::~GraphicsContextGLANGLE()
             bool result = EGL_DestroySync(m_displayObj, sync);
             ASSERT_UNUSED(result, !!result);
         }
-        EGL_MakeCurrent(m_displayObj, EGL_NO_SURFACE, EGL_NO_SURFACE, EGL_NO_CONTEXT);
         EGL_DestroyContext(m_displayObj, m_contextObj);
```

Comment:
Don't we need to keep that to make sure the context is not current when destroying it?